Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml

```
@@ -17,8 +17,9 @@
     ClothingHeadHatPwig: 2
     ClothingOuterRobesJudge: 2
     ClothingOuterPoncho: 2
-    ClothingFairyWings: 4 #sunrise-edit
-    ClothingLosyashWings: 1 #sunrise-edit
+    ClothingOuterKigurumiPanda: 4 #sunrise-edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)